toolchain/check/testdata/class/inheritance_access.carbon

```diff
@@ -178,16 +178,53 @@ class B {
   }
 
   fn SomeFunc[self: Self]() -> i32{
-    // CHECK:STDERR: fail_non_inherited_access.carbon:[[@LINE+6]]:12: ERROR: Cannot access protected member `INTERNAL_CONSTANT` of type `Internal`.
+    // CHECK:STDERR: fail_non_inherited_access.carbon:[[@LINE+7]]:12: ERROR: Cannot access protected member `INTERNAL_CONSTANT` of type `Internal`.
     // CHECK:STDERR:     return self.internal.INTERNAL_CONSTANT;
     // CHECK:STDERR:            ^~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
     // CHECK:STDERR: fail_non_inherited_access.carbon:[[@LINE-30]]:17: The protected member `INTERNAL_CONSTANT` is defined here.
     // CHECK:STDERR:   protected let INTERNAL_CONSTANT: i32 = 5;
     // CHECK:STDERR:                 ^~~~~~~~~~~~~~~~~
+    // CHECK:STDERR:
     return self.internal.INTERNAL_CONSTANT;
   }
 }
 
+// --- fail_compound_member_access.carbon
+
+library "[[@TEST_NAME]]";
+
+base class A {
+  private var x: i32;
+}
+
+class B {
+  fn F[self: Self]() {
+    // CHECK:STDERR: fail_compound_member_access.carbon:[[@LINE+6]]:11: ERROR: Cannot access private member `x` of type `A`.
+    // CHECK:STDERR:     self.(A.x);
+    // CHECK:STDERR:           ^~~
+    // CHECK:STDERR: fail_compound_member_access.carbon:[[@LINE-8]]:15: The private member `x` is defined here.
+    // CHECK:STDERR:   private var x: i32;
+    // CHECK:STDERR:               ^~~~~~
+    self.(A.x);
+  }
+}
+
+// --- inherited_compound_member_access.carbon
+
+library "inherited_compound_member_access";
```

Comment:
```suggestion
library "[[@TEST_NAME]]";
```

Reply:
Note, I was only looking for issues with library directives due to the recent change.